### PR TITLE
Abhishinde24

### PR DESCRIPTION
**Pull request by abhishinde24**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1555"></a>

### CODE REVIEW

**Comment by abhishinde24** [comment]
adding profile

**Review by Kartik2220 — CHANGES_REQUESTED** [review]
kindly add your avatar....

File: public/data/abhishinde24.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      {
+        "title": "Started DSA,CP,Machine learing,",
+        "date": "June 2021",
+        "icon": "code",
+        "color": "#90ee90",
+        "description": "Began Practicing Data structures and Algorithm, Competitive programing and Machine learning side projects",
+        "url": "#"
+      }
+    ]
+  }
```
Comment (aparna2071) [inline public/data/abhishinde24.json:28]: Kindly add a new line at the end(line 29) so that there is no Prettier issue.

**Review by aparna2071 — CHANGES_REQUESTED** [review]

File: public/data/abhishinde24.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started DSA,CP,Machine learing,",
+        "date": "June 2021",
+        "icon": "code",
+        "color": "#90ee90",
+        "description": "Began Practicing Data structures and Algorithm, Competitive programing and Machine learning side projects",
+        "url": "#"
```
Comment (kumarsonsoff3) [inline public/data/abhishinde24.json:25]: Kindly add your URL here, If you don't have a URL you can simply remove the line no `25`

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
**Will be Good to go after these changes!**

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**